challenges/Messaging/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 add_compile_options( -fno-exceptions -fno-rtti -DCPLUSPLUS )
 set( SERVICE_ID "00075" )
 set( AUTHOR_ID "KPRCA" )
-add_compile_options( -O3 -g -DDISABLE_HEAP_GUARD )
+add_compile_options( -O3 -g -DDISABLE_HEAP_GUARDi -fpermissive -fms-extensions )
```

Comment:
I think there's an extraneous `i` character at the end of `-DDISABLE_HEAP_GUARDi`

Reply:
my bad